### PR TITLE
Update breadcrumb for staff dashboard

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/21789

*Description of changes:
- Update breadcrumb view for staff dashboard
- Move the redirect logic for digital business card to the composable.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-33-hpcx83gz.web.app

File: src/app/router.options.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export default <RouterConfig> {
       meta: {
         layout: 'business',
         title: 'Business Dashboard',
-        breadcrumbs: [getBcrosHomeCrumb, getRegistryDashCrumb, getBusinessDashCrumb]
+        breadcrumbs: [getBcrosHomeCrumb, getRegistryDashCrumb, getBusinessDashCrumb],
+        staffBreadcrumbs: [getStaffDashCrumb, getBusinessDashCrumb]
```
Comment (hfekete) [inline src/app/router.options.ts:15]: Perfect thanks, now we have config for staff dash too :)

File: src/components/bcros/businessDetails/LinkActions.vue [hunk 2]
```diff
@@ -25,7 +24,7 @@ const allActions: ComputedRef<Array<MenuActionItem>> = computed(() => {
       disabled: false,
       label: t('button.tombstone.menuAction.digitalCredentials'),
       click: () => {
-        redirectToDigitalCredentialsPage()
+        goToDigitalCredentialsPage()
```
Comment (hfekete) [inline src/components/bcros/businessDetails/LinkActions.vue:27]: thank you for moving this to the composable!

File: src/layouts/default.vue [hunk 3]
```diff
@@ -15,7 +15,15 @@
 
 <script setup lang="ts">
 const route = useRoute()
-const crumbConstructors = computed(() => (route?.meta?.breadcrumbs || []) as (() => BreadcrumbI)[])
+const { isStaffAccount } = useBcrosAccount()
+
+const crumbConstructors = computed(() => {
```
Comment (hfekete) [inline src/layouts/default.vue:20]: just for the future reference you could simplify code a bit more like this:
```
const breadcrumbs = (isStaffAccount ? route?.meta?.staffBreadcrumbs : route?.meta?.breadcrumbs)
return (breadcrumbs || []) as (() => BreadcrumbI)[]
```

**Review by hfekete — APPROVED** [review]
Nice, thanks for all the updates!

**Pull request merged**